smdebug/core/hook.py

```diff
@@ -364,6 +365,14 @@ def _get_collections_with_tensor(self, tensor_name) -> Set["Collection"]:
     def _get_default_collections(self):
         pass
 
+    def has_default_hook_configuration(self):
```

Comment:
this will be called by framework code I assume?

Reply:
Yes, I have added comments.